post.html

```diff
@@ -5,8 +5,21 @@
   <!-- https://staticpub.mauve.moe/helloworld.html -->
   <!-- hyper://staticpub.mauve.moe/helloworld.html -->
   <!-- ipfs://bafybeifslnipwp5uanmhkckokwuse7h5gfrrjzqq4jg5oxewxbzrdcdawu -->
-  <distributed-post url="ipns://staticpub.mauve.moe/helloworld.html">
-  </distributed-post>
+  <distributed-post id="distributedPost"></distributed-post>
 </div>
 
 <script type="module" src="./post.js"></script>
+<!-- Dynamically set post URL from query string parameter -->
+<script>
+  document.addEventListener("DOMContentLoaded", function () {
```

Comment:
Does it work if you remove this? Usually the dom should be ready by the time the script is evaluated.

Reply:
Yes, it works.